.github/workflows/on-push-to-main-branch.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         uses: momentohq/standards-and-practices/github-actions/generate-and-commit-oss-readme@gh-actions-v1
         with:
           project_status: official
-          project_stability: beta
+          project_stability: stable
```

Comment:
there's one of these in on-pull-request too.  it's not crucial that they match, it's just making sure the readme gen isn't broken, but I have been keeping them updated together.

Reply:
Woops! I totally missed that. Fixed in https://github.com/momentohq/client-sdk-python/pull/309/commits/97dc7db46c27f1756bca19f5faed3f5228f8902c